native/Avalonia.Native/src/OSX/AvnPanelWindow.h

```diff
@@ -0,0 +1,9 @@
+//
+// Created by Dan Walmsley on 06/05/2022.
```

Comment:
1. I saw in the past some such headers were actually removed. I'm curious why these are being added back and if it's a new standard?
2. Why is it assigning copywrite to an individual instead of the project or foundation? That might actually not match with the contributor license agreement of Avalonia. Not sure.

Reply:
I believe it's autogenerated by xcode